.jenkinsci/text-variables.groovy

```diff
@@ -200,6 +200,17 @@ cmd_description = """
          </li>
       </ul>
    </li>
+   <li>
+      <p><strong>build_shared_libs</strong> = false&nbsp;</p>
+      <ul>
+         <li>
+            <p>builds libraries as shared libraries</p>
+         </li>
+         <li>
+            <p>Ex:&nbsp;build_shared_libs = true</p>
```

Comment:
```suggestion
            <p>Ex:&nbsp;build_shared_libs = true; x64linux_compiler_list= ['gcc5']; testing=true; testList = '()'</p>
```

Reply:
`x64linux_compiler_list= ['gcc5']` should not be a requirement, as well as `testing=true`